### ISSUE TITLE
How to train on 512*512 images with CRA?

### ISSUE THREAD

**Issue by renleidewenming** [Atlas200dk/sample-imageinpainting-HiFill#7]
Dear author, in the paper, the model was trained on 512x512 images. However, both the input and output of the generator are 512x512, which I think there is no need to resize the output to a high resolution to get the residuals. So my question is how do you train your CRA module on 512x512 images? Even there is no weights to be trained in CRA, it does affect the other part of the generator. And is the loss calculated on the generator output? Or the postprocess output (end-to-end training)?

**Comment by duxingren14** [comment]
loss is calculated on the generator outputs. Please find more detils here

https://github.com/duxingren14/Hifill-tensorflow
